<?xml version='1.0' encoding='UTF-8' ?>
<testsuite errors="0" failures="2" hostname="ASDAs-Mac-Pro.local" name="DummyObjCAllFailuresTests" tests="2" time="0.002" timestamp="2016-02-04 09:12:33 -0800"><testcase classname='DummyObjCAllFailuresTests' name='testFailure1' time='0.001'>
<failure message='failed - Failure 1' type='Failure'>/Users/Shared/Jenkins/Home/jobs/dwsjoquist testing/workspace/ASDA-Tests/Common/DummyObjCAllFailuresTests.m:19</failure>
</testcase>
<testcase classname='DummyObjCAllFailuresTests' name='testFailure2' time='0.001'>
<failure message='((@"wrong answer") equal to (a)) failed: ("wrong answer") is not equal to ("testFailure2")' type='Failure'>/Users/Shared/Jenkins/Home/jobs/dwsjoquist testing/workspace/ASDA-Tests/Common/DummyObjCAllFailuresTests.m:25</failure>
</testcase>
</testsuite>
